easybuild/tools/run.py

```diff
@@ -148,9 +151,9 @@ def run_cmd(cmd, log_ok=True, log_all=False, simple=False, inp=None, regexp=True
         runLog = None
 
     readSize = 1024 * 8
-
+    _log.info('running cmd: %s ' % cmd)
     try:
-        p = subprocess.Popen(cmd, shell=True, stdout=subprocess.PIPE, stderr=subprocess.STDOUT,
+        p = subprocess.Popen(cmd, shell=shell, stdout=subprocess.PIPE, stderr=subprocess.STDOUT,
```

Comment:
ok, I worry that it will have side effects, but it should be backwards compatible anyway.